docs/uptime/settings.asciidoc

```diff
@@ -25,3 +27,20 @@ to manually query Elasticsearch for data outside this pattern!
 See the <<uptime-security,Uptime Elasticsearch Security>>
 and {heartbeat-ref}/securing-heartbeat.html[Heartbeat security]
 docs for more information.
+
+=== Certificate thresholds
+
+The settings page also allows you to specify thresholds for all the TLS certificates monitored
+in your environment. There is a field for maximum age, and the minimum number of days before
```

Comment:
I don't think we need to be this descriptive, that part of the app is self explanatory. IMHO we shouldn't document the obvious. Would like @bmorelli25 's thoughts here.

Perhaps the content would be better if it described what the fields vs. documenting their existence. Rephrasing this as: "Use the certificate maximum age field to control the number of days before expiration Uptime should display the certificate as being in a warning state. Warnings for maximum certificate age can also be set here. Some user agents and browsers consider certificates invalid after a certain amount of time.

Reply:
Agreed. But looking at the screenshot, I only see "Expiration threshold" and "Age limit" as options. The docs should match those names. I'd alter Andrew's recommendation to something like this:

The settings page also allows you to specify thresholds for displaying and alerting on TLS certificates. Use the expiration threshold to specify <<STUFF>>, and the age limit field to control the <<STUFF>>. Some user agents and browsers consider certificates invalid after a certain amount of time.